packages/atclient/src/atclient_get_atkeys.c

```diff
@@ -46,7 +46,7 @@ int atclient_get_atkeys(atclient *atclient, atclient_atkey **atkey, size_t *outp
 
   char scan_cmd[scan_cmd_size];
 
-  const size_t recv_size = 8192; // TODO change using atclient_connection_read which will handle realloc
+  const size_t recv_size = 16384; // TODO change using atclient_connection_read which will handle realloc
```

Comment:
Increased this because scan results can be large, the TLS socket now creates a heap allocated buffer, but read and send still expect pre-allocated buffers. I didn't have time to uptake this in existing code yet. If the buffer in the heap is larger than the preallocated size, then the functions log and return an error code.